Formula/r/rustcat.rb

```diff
@@ -0,0 +1,26 @@
+class Rustcat < Formula
+  desc "Modern Port listener and Reverse shell"
+  homepage "https://github.com/robiot/rustcat"
+  url "https://github.com/robiot/rustcat/archive/refs/tags/v3.0.0.tar.gz"
+  sha256 "59648e51ab41e4aeb825174dfbb53710207257feb3757521be98ed28c4249922"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```

Comment:
We don't usually need an explicit `livecheck` block unless livecheck gives incorrect results.

```suggestion
```